administrator/components/com_associations/views/association/view.html.php

```diff
@@ -147,11 +147,28 @@ protected function addToolbar()
 		$input = JFactory::getApplication()->input;
 		$input->set('hidemainmenu', 1);
 
+		$user       = JFactory::getUser();
+		$userId     = $user->id;
+		$checkedOut = !($this->table->checked_out == 0 || $this->table->checked_out == $userId);
```

Comment:
@jreys DON'T USE direct table fields!! You can't be ever sure what is the field name of the 3º party component for created_by.

That's why i created this: https://github.com/joomla-projects/gsoc16_improved-multi-lingual/blob/staging/administrator/components/com_associations/helpers/associations.php#L214-L231

Please use it!